seed/seed.json

```diff
@@ -323,6 +323,34 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "AdServingIncreaseMaximumAdNotificationsPerDayTo60ForTier1And2Regions",
+            "experiments": [
+                {
+                    "name": "AdServing",
```

Comment:
I'd suggest we use "Enabled" or something similar as the experiment name

this is how it looks on brave://version
```
DoNotShowCustomAdNotificationsOnBetaAndReleaseChannelsStudy:AdNotifications
EphemeralStorageStudy:Enabled
PermissionLifetimeStudy:Enabled
```

Reply:
fixed as requested by @tmancey